__tests__/routes/web/searchResults.spec.ts

```diff
@@ -0,0 +1,549 @@
+/**
+ * @jest-environment jsdom
+ */
+
+'use strict';
+
+import { Server, ServerInjectResponse } from '@hapi/hapi';
+
+import { initializeServer } from '../../../src/infrastructure/server';
+import supertest from 'supertest';
```

Comment:
why are you introducing a new library now? Has this been approved by somebody?

Reply:
About the supertest package. Using server.inject takes a lot of time and causes a request timeout error while testing the POST request. Also, the assertions placed for POST request pass even if they are not correct. So I included this to be used in the test cases.